P2047-alloc-aware-optional.html

```diff
@@ -590,179 +569,181 @@ <h2>Proposed wording</h2>
 
 <blockquote><ins><code>
 namespace pmr {
-<br/><tab/>  // [alloc-optional.optional], class template <t><i>alloc-optional</i></t> for exposition only
+<br/><tab/>  // [pmroptional.optional]
 <br/><tab/>   template&lt;class T&gt;
-<br/><tab2/>     class <i>alloc-optional</i>;
-<br/>
+<br/><tab2/>     class optional;
+<br/>		  
```

Comment:
This says [optional.relops] and a whole bunch of [optional.*], aren't these supposed to be [pmroptional.*]?

Reply:
yup, fixed